### PR TITLE
Removed config sync logic for OS

### PR DESCRIPTION
**Pull request by bvtejaswi**
### :nut_and_bolt: Description: What code changed, and why?
Removed config sync logic for OS which is not needed

### :chains: Related Resources
https://chefio.atlassian.net/browse/CHEF-5662

### :+1: Definition of Done
After AWS Deployment did the following things:
-  Changed the max-shard-per-node value and patched to OS node
- Added new OS node
- New OS node has updated config
- upgarded with cli
- After upgradation it still has updated config

### :athletic_shoe: How to Build and Test the Change
build components/automate-cli

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
[Demo-video](https://progresssoftware.sharepoint.com/:v:/s/ChefCoreC/ERz0gqXPFxBFtwD2wulL2HAB8cu49VaD1Z9_IqX-3M6QqA?e=hp89Uz)

### CODE REVIEW

**Review by punitmundra — APPROVED** [review]

File: components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go [hunk 1]
```diff
@@ -750,22 +750,14 @@ func getNodeObjectsToFetchConfigFromAllNodeTypes() []*NodeObject {
 func getNodeObjectsToPatchWorkspaceConfigToAllNodes() []*NodeObject {
 	timestamp := time.Now().Format("20060102150405")
 	fmt.Println("====================================================================")
-	fmt.Println("sync Config to all nodes")
+	fmt.Println("sync Config to frontend nodes")
```
Comment (ArvinthC3000) [inline components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go:753]:
```suggestion
	fmt.Println("Syncing configs to frontend nodes")
```

File: components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go [hunk 2]
```diff
@@ -821,7 +808,7 @@ func newNodeTypeCmd(nodeMap *NodeTypeAndCmd, cmdString string, outputFiles []str
 	}
 }
 
-// prePatchForFrontendNodes parse frontend config before patch and remove cert related keys-values
+// prePatchForFrontendNodes parse frontend config before patch and remove cert related keys-values.
```
Comment (ArvinthC3000) [inline components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go:811]:
```suggestion
// prePatchForFrontendNodes parse frontend config before patch and remove cert related keys-values
```

File: components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go [hunk 3]
```diff
@@ -833,6 +820,7 @@ func prePatchForFrontendNodes(inputs *CmdInputs, sshUtil SSHUtil, infra *Automat
 	return nil
 }
 
+// prePatchForOsNodes is not being in used currently as it was meant for os-config sync
```
Comment (ArvinthC3000) [inline components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go:823]:
```suggestion
// prePatchForOsNodes 
// Note: this func is not being in used currently as it was meant for os-config sync
```

File: components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go [hunk 4]
```diff
@@ -844,6 +832,7 @@ func prePatchForOsNodes(inputs *CmdInputs, sshUtil SSHUtil, infra *AutomateHAInf
 	return nil
 }
 
+// removeRestrictedKeysFromSrcFileOs is not being in used currently as it was meant for os-config sync
```
Comment (ArvinthC3000) [inline components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go:835]:
```suggestion

```

File: components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go (hunk 1 above)
Comment (bvtejaswi) [reply components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go]: done

File: components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go (hunk 3 above)
Comment (bvtejaswi) [reply components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go]: done

File: components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go (hunk 4 above)
Comment (bvtejaswi) [reply components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go]: done

**Review by ArvinthC3000 — APPROVED** [review]

**Review by jayvikramsharma1 — APPROVED** [review]

**Pull request merged**